pynblint/nb_linting.py

```diff
@@ -152,6 +156,19 @@ def missing_closing_MD_text(notebook: Notebook) -> bool:
     )
 
 
+def duplicate_notebook_not_renamed(notebook: Notebook) -> bool:
+    """Check if the duplicate notebook has not been renamed.
+
+    I.e., if it was left with the default title:
+    ``<source-notebook-name>-Copy<copy-number>.ipynb``.
+    """
+    res = False
+    pattern: Pattern[str] = re.compile(r".*-Copy\d+.ipynb")
+    if pattern.match(notebook.path.name):
+        res = True
+    return res
```

Comment:
This change is related to a different issue.

Please, create a new feature branches from `main`, not from another feature branches.
Otherwise diffs are harder to read and merges might arise problems.

Reply:
Now I guess that you need to merge your PRs in the right chronological order.